### PR TITLE
docs: <Checkbox> Usage guidelines - fix font #1577

### PR DESCRIPTION
**Pull request by SoumyadiptoPal**
Fixes issue #1577 
#### Basic

- [x] Used plop (`npm run plop`) to create a new component.
- [x] PR has description.
- [x] New component is functional and uses Hooks.
- [x] Component is written in TypeScript.

#### Style

- [x] Styles are added to `NewComponent.modules.scss` file inside the `NewComponent` folder.
- [x] Component uses CSS Modules.
- [x] Design is compatible with [Monday Design System](https://design.monday.com/).
- [x] Styles are defined using [monday-ui-style](https://github.com/mondaycom/monday-ui-style) tokens
- [x] Displays correctly in all the themes

#### Storybook

- [x] Stories were added to `/src/components/NewComponent/__stories__/NewComponent.stories.mdx` file.
- [x] Stories include all flows of using the component.
- [x] Stories implemented using [vibe-storybook-components](https://github.com/mondaycom/vibe-storybook-components) components and tokens.

#### Tests

- [x] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

#### Accessibility

- [x] Component is accessible.
- [x] Component is compatible with [Vibe Accessibility Guidelines](https://style.monday.com/?path=/docs/foundations-accessibility--page)
image

### CODE REVIEW

**Comment by SoumyadiptoPal** [comment]
@SergeyRoyt I have fixed issue #1577

File: src/components/Checkbox/__stories__/checkbox.stories.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import "~vibe-storybook-components/dist/mixins";
+@import "~monday-ui-style/dist/mixins";
```
Comment (SergeyRoyt) [inline src/components/Checkbox/__stories__/checkbox.stories.scss:2]: It's not needed here - no mixins from monday-ui-style are used

File: src/components/Checkbox/__stories__/checkbox.stories.scss [hunk 2]
```diff
@@ -37,12 +38,13 @@
   &_list {
     display: flex;
     flex-direction: column;
+    font-family :var(--font-family) !important;
```
Comment (SergeyRoyt) [inline src/components/Checkbox/__stories__/checkbox.stories.scss:41]: `font-family` is not the only thing to define the font. Let's instead use `sb-content-text` mixin and increase the specificity of selector, so all the styles would apply. Btw, even thought, it's only documentation using `!important` in general is a bad practice

I propose we'd do smth like this:
```suggestion
    span {
      @include sb-content-text;
    }
```


And also we'll need to wrap `Use checkboxes to:` text with `span` tag
This way all out mixin styles will be applied both to the header title and to the list items without using `!important`

File: src/components/Checkbox/__stories__/checkbox.stories.scss [hunk 3]
```diff
@@ -37,12 +38,13 @@
...
   }
 
   &_list-item {
     display: inline-block;
     width: 100%;
-    @include sb-content-text();
+    font-family :var(--font-family) !important;
```
Comment (SergeyRoyt) [inline src/components/Checkbox/__stories__/checkbox.stories.scss:47]: Then this won't be needed too

**Review by SergeyRoyt — CHANGES_REQUESTED** [review]
Hey, @SoumyadiptoPal!
Thanks for the efforts! Left a few notes needed to finish the job! 💪

**Comment by SergeyRoyt** [comment]
Btw, it's always better to name the branches with some semantic names, e.g. in this case it could have been smth like
`docs/checkbox-usage-guidelines-font`
Doesn't really matter in this case though, no worries, just a tip for the future :)

**Comment by SoumyadiptoPal** [comment]
> Btw, it's always better to name the branches with some semantic names, e.g. in this case it could have been smth like `docs/checkbox-usage-guidelines-font` Doesn't really matter in this case though, no worries, just a tip for the future :)

Thank out for your advice. Will try to follow this from next time.👍👍

**Comment by SoumyadiptoPal** [comment]
@SergeyRoyt I have updated the code.

**Review by SergeyRoyt — APPROVED** [review]
Looks great, thanks!!
Please, also write a comment under the issue itself, so I can assign it to you and you'll get your Hactoberfest reward :)

**Pull request merged**